provision/minikube/keycloak/templates/keycloak.yaml

```diff
@@ -236,23 +233,6 @@ spec:
                 protocol: TCP
                 name: jvm-debug
 {{ end }}
-# tag::keycloak[]
-            resources:
-              requests:
-                {{ if .Values.cpuRequests }}cpu: "{{ .Values.cpuRequests }}"{{end}}
-                {{ if .Values.memoryRequestsMB }}memory: "{{ .Values.memoryRequestsMB }}M"{{end}}
-              limits:
-                {{ if .Values.cpuLimits }}cpu: "{{ .Values.cpuLimits }}"{{end}}
-                {{ if .Values.memoryLimitsMB }}memory: "{{ .Values.memoryLimitsMB }}M"{{end}}
-            # end::keycloak[]
-            # readinessProbe:
```

Comment:
do we want to leave the trace for probes for future use ?

Reply:
sure, we can keep that.